sweepbatcher/sweep_batcher.go

```diff
@@ -159,6 +160,15 @@ type VerifySchnorrSig func(pubKey *btcec.PublicKey, hash, sig []byte) error
 type FeeRateProvider func(ctx context.Context,
 	swapHash lntypes.Hash) (chainfee.SatPerKWeight, error)
 
+// PublishErrorHandler is a function that handles transaction publishing error.
+type PublishErrorHandler func(err error, errMsg string, log btclog.Logger)
+
+// logAllErrorsAsWarnings is an instance of PublishErrorHandler which logs all
+// errors as warnings.
+func logAllErrorsAsWarnings(err error, errMsg string, log btclog.Logger) {
+	log.Warnf("%s: %v", errMsg, err)
```

Comment:
nit: I think the default could be a little bit smarter too, like it could filter out obvious non-errors such as when it fails to RBF due to low fee, etc.

Reply:
I updated the function. Now it logs "insufficient fee" errors as info, everything else as a warning.